specs/linux/selinux_events.table

```diff
@@ -0,0 +1,11 @@
+table_name("selinux_events")
+description("Track SELinux events.")
+schema([
+    Column("type", TEXT, "Event type"),
```

Comment:
note, in `user_events` this is an Integer.

Reply:
I have renamed **msg** to **message**, but for the **type** column I'm using text labels; would you like me to change this? I could move the label to another column (like **type_name**) and change **type** back to integer